tests/complete/main.tf

```diff
@@ -142,10 +99,15 @@ module "eks_blueprints_addons" {
   enable_secrets_store_csi_driver              = true
   enable_secrets_store_csi_driver_provider_aws = true
   enable_kube_prometheus_stack                 = true
-  enable_external_dns                          = true
-  enable_external_secrets                      = true
-  enable_gatekeeper                            = true
-  enable_ingress_nginx                         = true
+
+  enable_external_dns = true
+  external_dns_route53_zone_arns = [
+    "arn:aws:route53:::hostedzone/*",
```

Comment:
pod was in crashloop because it was trying to list zones since we default to Route53